.docs/index.md

```diff
@@ -3,50 +3,24 @@
 
-```sh
-$ rexray volume --service virtualbox
-- attachments:
-  - instanceID:
-      id: e71578b0-1bfb-4fa5-bcd5-4ae982fd4a9b
-      driver: virtualbox
-    status: /Users/akutz/VirtualBox/libStorage/libStorage.vmdk
-    volumeID: 1b819454-a280-4cff-aff5-141f4e8fd154
-  name: libStorage.vmdk
-  size: 64
-  status: /Users/akutz/VirtualBox/libStorage/libStorage.vmdk
-  id: 1b819454-a280-4cff-aff5-141f4e8fd154
-  type: ""
-```
+Built on top of the [libStorage](http://libstorage.readthedocs.io/en/stable) framework, REX-Ray's simplified architecture consists of a single binary and runs as a stateless service on every host using a configuration file to orchestrate multiple storage platforms.
+
+![REX-Ray Architecture](http://i.imgur.com/LR9Hvig.gif)
 
-## Overview
-REX-Ray is a storage orchestration tool that provides a set of common commands
-for managing multiple storage platforms. Built on top of the
-[libStorage](http://libstorage.readthedocs.io/en/stable) framework, REX-Ray
-enables persistent storage for container runtimes such as Docker and
-Mesos.
+!!! note "note" 
 
-!!! note "note"
+    The current REX-Ray release omits support for several, previously verified storage platforms. These providers will be reintroduced incrementally, beginning with 0.4.1. If an absent driver prevents the use of REX-Ray, please continue to use 0.3.3 until such time the storage platform is re-introduced as a part of the [libStorage](http://libstorage.readthedocs.io/en/stable) framework. Instructions on how to [install](./user-guide/installation.md#rex-ray-033) and [configure](http://rexray.readthedocs.io/en/v0.3.3) REX-Ray 0.3.3 are both available.
 
-    The initial REX-Ray 0.4.x release omits support for several,
-    previously verified storage platforms. These providers will be
-    reintroduced incrementally, beginning with 0.4.1. If an absent driver
-    prevents the use of REX-Ray, please continue to use 0.3.3 until such time
-    the storage platform is introduced in REX-Ray 0.4.x. Instructions on how
-    to [install](./user-guide/installation.md#rex-ray-033) and
-    [configure](http://rexray.readthedocs.io/en/v0.3.3) REX-Ray 0.3.3 are both
-    available.
+## Supported Technologies
 
 ### Storage Provider Support
 The following storage providers and platforms are supported by REX-Ray.
 
 Provider              | Storage Platform(s)
 ----------------------|--------------------
+Amazon EC2 | [EFS](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers#aws-efs/)
```

Comment:
There is no change on the current master. I merged Travis's change onto `release/0.6.0` so it would get included in the pending release. For you I'll do the same. Please go ahead and make the adjustment @codenrhoden recommended, and then when I review the changes as well I will manually merge this PR into `release/0.6.0`.